backend/fregepoc/indexers/views.py

```diff
@@ -0,0 +1,11 @@
+from rest_framework import viewsets
+from rest_framework.permissions import DjangoModelPermissions
+
+from fregepoc.indexers.models import GitHubIndexer
+from fregepoc.indexers.serializers import GitHubIndexerSerializer
+
+
+class GitHubIndexerViewSet(viewsets.ModelViewSet):
+    queryset = GitHubIndexer.objects.all()
+    serializer_class = GitHubIndexerSerializer
+    permission_classes = [DjangoModelPermissions]
```

Comment:
Similarly as in the `GitHubIndexerSerializer` case - we don't want to expose the `Analyzer` & `Indexer` database instances via the API.